include/variables.h

```diff
@@ -3479,7 +3479,7 @@ extern GfxPool gGfxPools[2];
 extern u8 gAudioHeap[0x138000];
 extern u8 gSystemHeap[UNK_SIZE];
 
-extern u8 D_80780000[0x4600];
+extern u8 gPictoPhotoI8[0x4600];
```

Comment:
Maybe something else to think about: we could introduce defines for the resolution of the pictophoto here:
```
#define PICTO_RESOLUTION_X 160
#define PICTO_RESOLUTION_Y 112
```

But `z64snap.h` already uses this terminology for a different set of values, but that seems to be specifically for the range that a pictoactor will "register" as being in the photo, so I think that should be named after "CAPTURE_REGION"  or something.

Here's my first attempt at some of these names and macros:

https://github.com/engineer124/mm/commit/2d64b28e8ef3892f6ebe327d60b83441cb05d084